hex.pm.md

```diff
@@ -1,6 +1,6 @@
 # About Hex.pm Releases
 
-Releases of [esqlite on hex.pm](https://hex.pm/packages/esqlite) are managed by Michael Ries ([@mmmries](https://github.com/mmmries)) and Eric Scouten ([@scouten](https://github.com/scouten)).
+Releases of [esqlite on hex.pm](https://hex.pm/packages/esqlite) are managed by Connor Rigby ([@connorrigby](https://github.com/mmmries)).
```

Comment:
maybe just change that github link to be `github.com/connorrigby` 😄 . Thanks again for taking over. Super grateful for the quality you are bringing to these projects.